crowbar_framework/public/503_upgrade.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Error 503 | Crowbar
+    </title>
+
+    <meta charset="utf-8" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+    <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+
+    <link rel="stylesheet" href="/assets/vendor.css" />
+    <link rel="stylesheet" href="/assets/application.css" />
+    <link rel="stylesheet" href="/assets/branding.css" />
+
+    <script src="/assets/vendor.js"></script>
+    <script src="/assets/application.js"></script>
+    <script src="/assets/branding.js"></script>
+
+    <!--[if lt IE 9]>
+    <link rel="stylesheet" href="/assets/ie.css" />
+    <script src="/assets/ie.js"></script>
+    <![endif]-->
+  </head>
+  <body class="nodes">
+    <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+      <div class="container">
+        <div class="navbar-header">
+          <button class="navbar-toggle" data-target=".navbar-collapse" data-toggle="collapse">
+            <span class="sr-only">
+              Navigation
+            </span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a href="/" class="navbar-brand">
+            <span class="title">Crowbar</span>
+          </a>
+        </div>
+        <div class="collapse navbar-collapse navbar-right">
+          <ul class="nav navbar-nav">
+            <li>
+              <a href="/upgrade">Go to upgrade wizard</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="container" id="errorpage">
+      <div class="panel panel-default panel-warning">
+        <div class="panel-heading">
+          <h2>
+            The requested service is temporarily unavailable
+          </h2>
+        </div>
+        <div class="panel-body">
+          <p>
+            You are currently upgrading your cloud and therefore access to the Crowbar UI is
+            restricted to the <a href="/upgrade">upgrade wizard</a>.
+            Please finish the upgrade first.
+          </p>
+        </div>
+      </div>
```

Comment:
Hrm, can we add a HTML comment about this? Because I'm sure someone will add it in the future to make the page consistent with the other static pages.

(Btw, can't we have rails render other bits too?)